app/src/main/java/chat/rocket/android/settings/presentation/SettingsPresenter.kt

```diff
@@ -84,13 +85,15 @@ class SettingsPresenter @Inject constructor(
                         )
                     }
                 }
+                view.hideLoading()
```

Comment:
Not needed to call `hideLoading()` two times here. You can call it on the `finally {...}` block just one time instead.